packages/react-components/index.js

```diff
@@ -35,20 +35,37 @@ module.exports = (opts = {}) => (neutrino) => {
       neutrino.use(react(options));
     },
     () => {
-      const components = join(
-        neutrino.options.source,
-        options.components || 'components',
-      );
-
       Object.keys(neutrino.options.mains).forEach((key) => {
         delete neutrino.options.mains[key]; // eslint-disable-line no-param-reassign
       });
 
-      readdirSync(components).forEach((component) => {
-        // eslint-disable-next-line no-param-reassign
-        neutrino.options.mains[basename(component, extname(component))] = {
-          entry: join(components, component),
-        };
+      const componentPaths = [];
+      if (Array.isArray(options.components)) {
```

Comment:
I think we can simplify this slightly, by doing something like this psuedo-code:

```
if <options.components is null>
  set default value of `['components']

if <options.components is not an array>
  wrap current options.components value in an array

for each element in options.components
  // do stuff
```

I think we can ignore the empty array case, since it's a bit of an edge case, and it's not the default value, so if the user has passed an empty array, then it's an explicit choice to not set any components (although a nonsensical one).

Reply:
Hi, thank you for reviewing, I have done the changes and tests as requested. Let me know if there is anything I should change.